api/diagnosis/diagnosis_routes_v2.py

```
@@ -0,0 +1,87 @@
+
+def preprocess_input_image(frames: list) -> np.ndarray:
+    try:
+        processed_images = []
+        for frame in frames:
+            image = np.array(frame)
+            image = cv2.resize(image, (145, 145))
+            image = image / 255.0
+            processed_images.append(image)
+
+        input_array = np.array(processed_images)  # (N, 145, 145, 3)
+        return input_array
```

Ruff (0.11.9)

Consider moving this statement to an `else` block

(TRY300)